packages/stable/src/.cognite-openapi-snapshot.json

```diff
@@ -89658,8 +89658,7 @@
                     "x-additionalPropertiesName": "cursor-name",
                     "allOf": [
                       {
-                        "type": "string",
-                        "description": "The value to use for the cursor"
+                        "$ref": "#/components/schemas/NextCursorV3"
```

Comment:
Note: This is a forward fix for https://github.com/cognitedata/service-contracts/pull/2895 should not be merged until this PR is merged